src/world.rs

```diff
@@ -297,6 +297,12 @@ impl Generation {
         Generation(v)
     }
 
+    /// Returns the id of the generation.
+    #[inline]
+    pub fn get_id(&self) -> i32 {
```

Comment:
Hold on, the title says it's about generations, but this is the entity id, right?